8ball_responses.json

```diff
@@ -1,14 +1,26 @@
 {
-  "responses": [
-    "Yes",
-    "No",
-    "Maybe",
-    "Is my name TIJK-Bot?",
-    "Google it you twat",
-    "DEFINITELY",
-    "No, I am very sorry for you",
-    "I am not sure about that",
-    "I do not understand",
-    "What are you going to do if I don't answer?"
-  ]
-}
+    "responses": [
+      "Yes",
+      "No",
+      "Maybe",
+      "Is my name TIJK Bot Games?",
+      "Google it you twat",
+      "DEFINITELY",
+      "No, I am very sorry for you",
+      "I am not sure about that",
+      "I do not understand",
+      "What are you going to do if I don't answer?",
+      "I need to think about it.",
+      "Try again",
+      "1+1=3, which is true so to answer your question, yes",
+      "HELL NAH",
+      "Very doubtful",
+      "Sure",
+      "Without a doubt",
+      "Most likely",
+      "Might be possible",
+      "You'll be the judge",
+      "no... (╯°□°）╯︵ ┻━┻",
+      "might be yes, but who am i to judge?"
```

Comment:
Please fix grammar issues (capital letter at the start of a sentence, the "i" should be uppercase)